File: SecondYear/AvantikaSharma2307_AvantikaSharma_2226cseai1008_2/Week_2/factorial.py
from typing import *

def factorial(n: int) ->int:
    fact=1
    for i in range(1,n+1,1):
        fact=fact*i
    return fact



def factorialNumbers(n: int) -> List[int]:
    # Write your code here
    List=[]
    for i in range(1,n+1,1):
        if(factorial(i)<=n):
            List.append(factorial(i))
        else:
            break    
    return List

File: SecondYear/Parth27904_ParthAgarwal_2226CSE(AI&ML)1015_2/Week_2/factorial.java
import java.util.ArrayList;
import java.util.List;
public class factorial {
    public static List<Long> factorialNumbers(long n) {
        // Write Your Code Here
        List<Long> l1 = new ArrayList<Long>();
        long fact = 1;
        long i =1;
        while(fact<=n){
            l1.add(fact);
            i++;
            fact = fact*i;
        }
        return l1;
    }
}